src/components/TodoComponents/Todo.js

```diff
@@ -2,11 +2,11 @@ import React from 'react';
 
 import './Todo.scss';
 
-const Todo = ({item}) => {
+const Todo = ({item, clickHandler}) => {
 
   return (
 
-    <li>{item.task}</li>
+    <li onClick={() => clickHandler(item.id)} className={item.completed ? 'completed' : null}><input type='checkbox' onChange={() => {}} checked={item.completed} /> {item.task} <p></p></li>
```

Comment:
You should certainly move the attributes here onto separate lines for readability:
```javascript
<li 
  onClick={() => clickHandler(item.id)} 
  className={item.completed ? 'completed' : null}
>
  <input 
    type='checkbox' 
    onChange={() => {}} checked={item.completed} 
  /> 
  {item.task} 
  <p></p>
</li>
```
I had no idea what all was going on here until I broke up the tags and the attributes.

Reply:
You could also use object destructuring even moreso via:
```javascript
const { completed, id, task } = item;
```
and I assume you can imagine that the code that follows would be a bit shorter.